modules/core/test/test_mat.cpp

```diff
@@ -2354,6 +2354,95 @@ TEST(Mat, regression_18473)
     EXPECT_EQ((int)5, (int)m.at<short>(19, 49, 99));
 }
 
+// TEST(Mat1D, basic)
```

Comment:
In the test of core module, there is no skip flag like the one in dnn tests. So comment for now.

Reply:
You can add DISABLED_ to test name. It's compiled and skipped during default test execution.

Reply:
Thank you! Let me have a try.

Reply:
https://github.com/google/googletest/blob/main/docs/advanced.md#temporarily-disabling-tests

Reply:
Added prefix DISABLED_ to the test.